using-mobile-extensions/adobe-target/README.md

```diff
@@ -183,64 +182,223 @@ prefetchMboxesList.add(prefetchRequest2);
 
 
 // Call the prefetchContent API.
-Target.prefetchContent(prefetchMboxesList, profileParameters, prefetchStatusCallback);
+TargetParamters targetParameters = null;
+Target.prefetchContent(prefetchMboxesList, targetParameters, prefetchStatusCallback);
 ```
 {% endtab %}
 
 {% tab title="iOS" %}
 #### Objective C
 
-Use `prefetchContent` to send a prefetch request to your configured Target server with the `ACPTargetPrefetchObject` array and specified `profileParameters`. The callback will be invoked when the prefetch is complete, which returns a success status for the prefetch request.
+Use `prefetchContent` to send a prefetch request to your configured Target server with the `ACPTargetPrefetchObject` array and specified `ACPTargetParameters`. The callback is invoked when the prefetch is complete and, if successful, returns a nil value. If the prefetch is not successful, an error message is returned.
```

Comment:
- In the 1st sentence, please add "the" between "and" and "specified".
- In the 2nd sentence, is it supposed to be "null" or "nil"?

Reply:
it is supposed to be `nil` for iOS